Dockerfile.armv7l

```diff
@@ -0,0 +1,26 @@
+FROM rust:1.83.0-alpine3.20
+
+# Download the compiler
+RUN apk add wget
+RUN wget https://github.com/userdocs/qbt-musl-cross-make/releases/download/2450/armv7l-linux-musleabihf.tar.xz && \
```

Comment:
I'm honestly a little scared of https://github.com/userdocs. Do you have background info on why that's a reliable source for this pre-cross-compiled gcc?

I trust cross-rs a good bit more. They have these:
```
docker run --rm -it ghcr.io/cross-rs/armv7-unknown-linux-musleabihf:edge bash -c 'uname -m && arm-linux-musleabihf-gcc --version && echo "void main(){}" > main.c && arm-linux-musleabihf-gcc main.c -o main && file main'
x86_64
arm-linux-musleabihf-gcc (GCC) 9.2.0
Copyright (C) 2019 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

main: ELF 32-bit LSB shared object, ARM, EABI5 version 1 (SYSV), dynamically linked, interpreter /lib/ld-musl-armhf.so.1, not stripped
```
With all the Docker files that built them here: https://github.com/cross-rs/cross/tree/main/docker

Reply:
Ok, I'm trying out cross to set up ptex form aarch32 releases. I'll add you to the review when I have a PR ready. Hopefully later today. We can compare the approaches.

Reply:
Yeah that is much better to use `cross`. To be honest I just wanted to get something working, so I could submit the PR for feedback.